apps/dashboard/src/components/auth/questionnaire-form.tsx

```diff
@@ -9,7 +9,7 @@ import { useForm, Controller } from 'react-hook-form';
 import { updateClerkOrgMetadata } from '../../api/organization';
 import { hubspotCookie } from '../../utils/cookies';
 import { identifyUser } from '../../api/telemetry';
-import { useTelemetry } from '../../hooks';
+import { useTelemetry } from '../../hooks/use-telemetry';
```

Comment:
We should avoid the index.ts barrel file. It destroys tree shaking, chunking and introduces an extra import and file resolution.